components/measurement/StatusHeader.js

```diff
@@ -19,19 +20,19 @@ const StatusHeader = ({ status, color, icon, label, info }) => {
   if (status) {
     switch (status) {
     case 'anomaly':
-      computedLabel = 'Anomaly'
+      computedLabel = <FormattedMessage id='Measurement.Status.Header.Websites.Anomaly' />
```

Comment:
We should pick either `Header` or `Heading` (see other comment on `Headings`).

I am OK with either.

Reply:
I guess `Heading` is most consistent with our design system: https://github.com/ooni/design-system/blob/master/components/atoms/Heading.js

Reply:
The `Header` here refers to the status header region as opposed to headings as text elements. I have used `Headings` for such keys.

Reply:
Ah ok I see. Maybe we can call it `Hero` or `Jumbotron` (to take the boostrap terminology) to make it clear that it's a different thing to Headings (we use the `Hero` term on the mobile app).

Reply:
I guess I will change it in the file names and other references as well. It is better to follow consistent naming. Sorry about that.

Reply:
These are done.